### PR TITLE
Contributor attendance mode

### LINKED ISSUE #346: Add attendance mode to attendee list
Add icons for attendance modes to attendee/contributor list to identify the different attendance modes( remote, onsite).

### PR DESCRIPTION
**Pull request by trymebytes**
Fixes #346 In this PR, we add the video icon to the user avatar to identify remote attendees and contributors.

**Before**
Screenshot 2024-10-14 at 15 46 45

**After**
Screenshot 2024-10-14 at 15 47 01

### CODE REVIEW

**Review by amieiro — DISMISSED** [review]
I have tested at [WordCamp Granada](https://translate.wordpress.org/events/2024/wordcamp-granada/) (hybrid event) and at [WordCamp Kerala](https://translate.wordpress.org/events/2024/wordcamp-kerala-contributor-day/) (remote event) and the icons are not properly aligned. 

What do you think to add these CSS to a CSS file instead of adding them inline? 

image

image

**Comment by trymebytes** [comment]
Here is the updated view;

image

File: themes/wporg-translate-events-2024/style.css [hunk 1]
```diff
@@ -162,6 +162,33 @@ select.wp-block-form-input__input {
 	background-color: #46b450;
 }
 
+span.user-remote-icon.dashicons-video-alt2 {
+	margin-left: -8.8%;
+    margin-top: 32px;
```
Comment (akirk) [inline themes/wporg-translate-events-2024/style.css]: Mix of tabs and spaces here.

**Review by akirk — APPROVED** [review]
Tested and works well!

**Pull request merged**